### ISSUE TITLE
mp3's start with a delay and not instantly like WAVs

### ISSUE THREAD

**Issue by fred0r** [ValveSoftware/halflife#599]
Since the Update there's a delay with playing mp3's.

mp3 play "sound/test.mp3" -> delay
Playing the same file as WAV there's no delay.
A lot of AMX-Plugins use mp3s,

pls fix this !

**Comment by MatiasEsf** [comment]
in no-steam this delay no appear, but in steam version yes

**Comment by MatiasEsf** [comment]
Non-steam version has no delay when playing MP3 sound but the Steam client has a mini-delay when trying to play a MP3 sound

**Comment by SamVanheer** [comment]
The mp3 playback system is run on a separate thread but the code that sets up playback executes almost immediately, so it looks like this is a delay that occurs inside Miles itself. Perhaps a library upgrade introduced this issue?
